swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -4821,6 +4845,14 @@ components:
           description: Unique end to end identity.
           type: string
           maxLength: 35
+        batchIndicator:
+          description: |
+            If this indicator equals true, then the related entry is a batch entry.
+          type: boolean
+        batchNumberOfTransactions:
+          description: |
+            Shall be used if and only if the batchIndicator is contained and equals true.
+          type: integer
```

Comment:
Relevant Change: batchIndicator and batchNumberOfTransactions are new. We already have it
Adoption approach: Nothing to change

Reply:
already in there